src/api/stats/moonbeam/getStellaswapApys.js

```diff
@@ -1,25 +1,45 @@
+import { merge } from 'lodash';
 const { moonbeamWeb3: web3 } = require('../../../utils/web3');
 import { MOONBEAM_CHAIN_ID as chainId } from '../../../constants';
 import { getMasterChefApys } from '../common/getMasterChefApys';
 import { stellaClient } from '../../../apollo/client';
-const pools = require('../../../data/moonbeam/stellaswapLpPools.json');
+import { stellaswap } from '../../../../packages/address-book/address-book/moonbeam/platforms/stellaswap';
+const poolsV1 = require('../../../data/moonbeam/stellaswapLpPools.json');
+const poolsV2 = require('../../../data/moonbeam/stellaswapLpV2Pools.json');
 const getBlockTime = require('../../../utils/getBlockTime');
 
 const getStellaswapApys = async () =>
-  await getMasterChefApys({
-    web3: web3,
-    chainId: chainId,
-    masterchef: '0xEDFB330F5FA216C9D2039B99C8cE9dA85Ea91c1E',
-    tokenPerBlock: 'stellaPerBlock',
-    hasMultiplier: false,
-    secondsPerBlock: await getBlockTime(1284),
-    pools: pools,
-    oracleId: 'STELLA',
-    oracle: 'tokens',
-    decimals: '1e18',
-    tradingFeeInfoClient: stellaClient,
-    liquidityProviderFee: 0.0025,
-    // log: true,
-  });
+  merge(
+    await getMasterChefApys({
+      web3: web3,
+      chainId: chainId,
+      masterchef: stellaswap.masterchef,
+      tokenPerBlock: 'stellaPerBlock',
+      hasMultiplier: false,
+      secondsPerBlock: await getBlockTime(1284),
+      pools: poolsV1,
+      oracleId: 'STELLA',
+      oracle: 'tokens',
+      decimals: '1e18',
+      tradingFeeInfoClient: stellaClient,
+      liquidityProviderFee: 0.0025,
+      // log: true,
+    }),
+    await getMasterChefApys({
+      web3: web3,
+      chainId: chainId,
+      masterchef: stellaswap.masterchefV1distributorV2,
+      tokenPerBlock: 'stellaPerSec',
+      hasMultiplier: false,
+      secondsPerBlock: await getBlockTime(1284),
```

Comment:
this is not correct, APY will be much smaller, should probably be 1

Reply:
I did set `secondsPerBlock` to be 1, is this correct?
stellaswap shows 153% "annualized apr", after this fix I get this breakdown:
```
"stellaswap-ape-wglmr":{"vaultApr":1.4586396518347358,"compoundingsPerYear":2190,"beefyPerformanceFee":0.045,"vaultApy":3.2980185201588483,"lpFee":0.0025,"tradingApr":0,"totalApy":3.2980185201588483}
```
So vaultApr looks OK, but the stellaswap UI doens't show trading fees details so idk